TurkishFilter/sections/general_extensions.txt

```diff
@@ -32,6 +32,8 @@ technopat.net#%#(function() { var w_open = window.open, regex = /goo.gl/; window
 ! END: Clickunders
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/176425
+akakce.com#$#li.ignore { top: 0 !important; }
```

Comment:
It seems that unfortunately it doesn't work on my end.
Maybe just:
```adblock
akakce.com##li.ignore
```
will be fine?
Or more specific:
```adblock
akakce.com##li.ignore:has(> a.billboard_content)
```

By the way, if I'm not wrong, it looks like that this ad is displayed only if there is `window.plBannerState` object.